templates/buildkite-elastic.yml

```diff
@@ -109,6 +111,16 @@ Parameters:
     Type: Number
     Default: 0
 
+  ScaleOutAdjustment:
+    Description: The number of agents to add on each scale out event
+    Type: Number
+    Default: 5
+
+  ScaleInAdjustment:
+    Description: The number of agents to remove on each scale in event
```

Comment:
The number of agents to add on each scale out event ("ScheduledJobsCount > 0 for 1 minute")

The number of agents to remove on each scale in event ("ScheduledJobsCount == 0 for 5 minutes")